Bharat-sim-doc.rst

```diff
@@ -0,0 +1,45 @@
+.. Bharat-Sim documentation master file, created by
+   sphinx-quickstart on Mon Jul 12 19:47:20 2021.
+   You can adapt this file completely to your liking, but it should at least
+   contain the root `toctree` directive.
+
+Bharat-Sim's documentation!
+======================================
+
+.. toctree::
+   :maxdepth: 2
+   :caption: Contents:
+
+
+* :ref:`genindex`
+* :ref:`modindex`
+* :ref:`search`
+
+Overview
+==================
+
+BharatSim is a framework for scientific researchers to build an `agent based model <https://en.wikipedia.org/wiki/Agent-based_model>`_ to help envision the disease spread of COVID-19, devise, visualize and compare different interventions strategies (like vaccinations or lockdowns) and incorporate the effects of different levels of compliance. It allows the user to create hypothetical scenarios and simulate them. The agents in this model are normal people, going about their daily lives.
+
+BharatSim uses Scala, a language that uses the JVM or Java Virtual Machine to run. To easily view all the code, one can install IntelliJ, an IDE to view, modify and run the code.
+
+Installation
+=============
+
+Prerequisites
+--------------
+1. Setup scala and sbt - https://docs.scala-lang.org/getting-started/index.html#install-scala
```

Comment:
Make these hyperlinks as well

Reply:
Do you mean the titles like Overview, Installation?

Reply:
No, in the sense that this URL won't render as a URL unless you create mark up -- something like
`Setup scala and sbt <https://docs.scala-lang.org/getting-started/index.html#install-scala>`_ 

maybe ?

Reply:
Oh right, sure. So originally, the text 'Setup scala and sbt' and the link was separate (the link still works that way) but I can make the text into a URL.